plugins/source/mixpanel/docs/tables/mixpanel_export_events.md

```diff
@@ -0,0 +1,16 @@
+# Table: mixpanel_export_events
+
+The composite primary key for this table is (**project_id**, **time**).
```

Comment:
Would definitely be nice to make this table incremental, it could be a _lot_ of data